### PR TITLE
feat: add Form Fieldset UI component

### PR DESCRIPTION
**Pull request by rasokolovska**
## Description
Form Fieldset serves as a container for grouping related form controls, providing a gray frame with a heading to structure and organize form elements. Legend element, identified by the `form-fieldset__legend` class, specifies the purpose of the grouped elements.

## Task Link
[Figma Layout](https://www.figma.com/file/TYlLseXoMXDflJYbU1DGHa/HTML-2-%2F-%D0%9F%D0%B8%D0%BD%D0%BA?type=design&node-id=1226%3A599&mode=design&t=CExMh4Gf8c758dTo-1)

## Usage
Base class for the block is `form-fieldset`. Along with the mandatory base class, it requires the corresponding modifier:
-  `form-fieldset_for_personal-info`
- `form-fieldset_for_app-type`
- `form-fieldset_for_contact-info`
- `form-fieldset_for_achievements`
- `form-fieldset_for_impressions`

The correct settings for the legend element demand the base class `form-fieldset__legend` paired with the corresponding modifier:
- `form-fieldset__legend_for_app-type`
- `form-fieldset__legend_for_contact-info`
- `form-fieldset__legend_for_achievements`
- `form-fieldset__legend_for_impressions`

### Example of Usage
```html
<fieldset class="form-fieldset form-fieldset_for_achievements">
  <legend  class="form-fieldset__legend form-fieldset__legend_for_achievements">Ваши безбашенные достижения в путешествии</legend>
  <!-- form controls go here -->
</fieldset>
```

## Checklist
Before submitting this pull request, please ensure the following:

- [x] I've validated my HTML with [Nu Html Checker](https://validator.w3.org/nu/), and it outputs no errors
- [x] I've checked my HTML with [HTML Tree Generator](https://yoksel.github.io/html-tree/en/), and it outputs no errors
- I've ensured that the layout matches the mockup for the following control points:
  - [x] 1200px
  - [x] 660px
  - [x] 320px

### CODE REVIEW

File: blocks/form-fieldset/form-fieldset.css [hunk 1]
```diff
@@ -1,5 +1,4 @@
 .form-fieldset{
-  border: rgb(229, 229, 229) solid 2px;
```
Comment (MarPostovik) [inline blocks/form-fieldset/form-fieldset.css]: @rasokolovska  why did you remove this line?

**Review by naumch1k — APPROVED** [review]
